core/src/utils/browser/index.ts

```diff
@@ -72,7 +72,32 @@ type IonicEvents = {
   ): void;
 };
 
-type IonicWindow = Window & IonicEvents;
+export interface CloseWatcher extends EventTarget {
+  new (options?: CloseWatcherOptions): any;
```

Comment:
Having this return `CloseWatcher` gives me the following lint error:

```
/Users/liamdebeasi/Ionic/ionic/core/src/utils/browser/index.ts
  76:3  error  Interfaces cannot be constructed, only classes  @typescript-eslint/no-misused-new

✖ 1 problem (1 error, 0 warnings)
```

However, using `any` still gives us type checking:

```
[ ERROR ]  TypeScript: src/utils/hardware-back-button.ts:120:16
           Property 'liam' does not exist on type
           'CloseWatcher'.

    L119:   */
    L120:  watcher!.liam = true;
    L121:  watcher!.requestClose();
```